lib/structures/GuildEvent.js

```diff
@@ -0,0 +1,129 @@
+"use strict";
+
+const Base = require("./Base");
+
+/**
+* Represents a guild scheduled event
+* @prop {String} id The id of the guild event
+* @prop {String} guildID The guild id of the event
+* @prop {String} channelID The channel id of the event
+* @prop {String} name The name of the event
+* @prop {String?} description The description of the event
+* @prop {String} image The image hash of the event
+* @prop {Date} scheduledStartTime The date the event will start
+* @prop {Date} scheduledEndTime The time the event will end, or null if the event does not have a scheduled time to end
+* @prop {Number} privacyLevel Event privacy level
+* @prop {Number} status The scheduled status of the event
+* @prop {Number} entityType The scheduled entity type of the event
+* @prop {String} entityID Entity id
+* @prop {Object} entityMetadata Metadata for the event
+* @prop {Array<String>} entityMetadata.speakerIDs The speakers of the stage channel
+* @prop {String} entityMetadata.location Location of the event
+* @prop {Array<String>} skuIDs Sku ids
+* @prop {Array} skus Skus
```

Comment:
This has no meaning

Reply:
Currently, discord only documents "skus" as Skus, and also gives a type of array, but doesn't provide the value of the array.